app/src/main/java/com/example/coinage/LoginActivity.java

```diff
@@ -17,11 +26,18 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_login);
 
+        etEmail = findViewById(R.id.etEmail);
+        etPassword = findViewById(R.id.etPassword);
+
         btnLogin = findViewById(R.id.btnLogin);
         btnLogin.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                goHomeActivity();
+                String email = etEmail.getText().toString();
+                String password = etPassword.getText().toString();
+                Log.i(TAG, email);
+                Log.i(TAG, password);
```

Comment:
logging passwords is generally an unsafe security practice .. just fyi!

Reply:
good point!